scripts/wrappers/gatk/gatkToWdlWrapper.py

```diff
@@ -0,0 +1,210 @@
+                                      "} \\\n")
+                    else:
+                        wdlFile.write("\t\t\t" + engineData[toolName][b][0] + " ${default=\"" + checkQuotes(engineData[toolName][b][1]) + "\" " +
+                                      checkName(engineData[toolName][b][0][2:]) + "} \\\n")
+                else:
+                    print("There was an error determining the optional status of an engine argument: Input recieved: " +
+                          engineData[toolName][b][2] + " for the argument " + toolName + ": " + engineData[toolName][b][0])
+                    
+
+        #iterate through remaining options (non-defaults)
+        for b in range(0,len(toolData["parallel"])):
+            wdlFile.write("\t\t\t${default=\"\" \"" + toolData["parallel"][b]["arg"] + "\" + " + toolData["parallel"][b]["arg"][1:] + "Val} \\\n")                 
+        for b in range(0,len(toolArgs)):
+            if toolArgs[b]["required"] == "yes":
+                wdlFile.write("\t\t\t" + toolArgs[b]["synonyms"] + " ${" + checkName(toolArgs[b]["name"][2:]) + "} \\\n") 
+            elif toolArgs[b]["required"] == "no":
+                if toolArgs[b]["defaultValue"] != "NA" and toolArgs[b]["defaultValue"] != "none" and toolArgs[b]["defaultValue"] != "[ ]":
+                    if toolArgs[b]["synonyms"] != "NA":
+                        wdlFile.write("\t\t\t" + toolArgs[b]["synonyms"] + " ${default=\"" + checkQuotes(toolArgs[b]["defaultValue"]) + "\" " +
+                                  checkName(toolArgs[b]["name"][2:]) + "} \\\n")
+                    else:
+                        wdlFile.write("\t\t\t" + checkName(toolArgs[b]["name"][2:]) + " ${default=\"" + checkQuotes(toolArgs[b]["defaultValue"]) + "\" " +
+                                  checkName(toolArgs[b]["name"][2:]) + "} \\\n")
+                else:
+                    if toolArgs[b]["synonyms"] != "NA":
+                        wdlFile.write("\t\t\t${default=\"\" \"" + toolArgs[b]["synonyms"] + " \" + " + checkName(toolArgs[b]["name"][2:]) + "} \\\n")
+                    else:
+                        wdlFile.write("\t\t\t${default=\"\" \"" + checkName(toolArgs[b]["name"][2:]) + " \" + " + checkName(toolArgs[b]["name"][2:]) + "} \\\n")
+            else:
+                print("There was an error determining optional status of an argument. Input received: " +
+                      toolArgs[b]["required"] + " for the argument " + toolArgs[b]["synonyms"])
+        wdlFile.write("\t}\n")
+
+        #write output
+        wdlFile.write("\n\toutput {\n" +
+                      "\t\t#To track additional outputs from your task, please manually add them below\n" +
+                      "\t\tString taskOut = \"${out}\"\n" +
+                      "\t}\n")
+
+        #write runtime
+        #TODO Replace docker image with public one
```

Comment:
The docker below is public, isn't it? Do you mean replace it with a GATK-only docker once we have that? (which I should ping Brad on)